### PR TITLE
Fixes #12923: Move SimplePackage to pulp/service object

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: app/services/katello/pulp/simple_package.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+module Katello
+  module Pulp
+    class SimplePackage
+      # {"epoch", "name", "arch", "version", "vendor", "release"}
```
Comment (daviddavis) [inline app/services/katello/pulp/simple_package.rb]: What's this for? Mind removing it if it's no longer needed?

**Comment by daviddavis** [comment]
ACK. I don't care if you address my comment or not.

**Comment by ehelms** [comment]
[test]

File: app/services/katello/pulp/simple_package.rb (hunk 1 above)
Comment (ehelms) [inline app/services/katello/pulp/simple_package.rb]: This class? or the comment?

**Pull request merged**